tests/rbd_mirror/test_rbd_mirror_daemon_status.py

```diff
@@ -0,0 +1,54 @@
+from tests.rbd_mirror import rbd_mirror_utils as rbdmirror
+from utility.log import Log
+
+log = Log(__name__)
+
+
+def run(**kw):
+    """Verification rbd mirror and daemon status on cluster
+
+    Args:
+        **kw: Key/value pairs of configuration information to be used in the test.
+    Returns:
+        0 - if test case pass
+        1 - it test case fails
+    Test case covered : CEPH-83573760
+    Test Case Flow:
+    1. Deploy two clusters using Cephadm
+    2. Configures RBD Mirroring
```

Comment:
Since we already have option to configure RBD mirroring in cephci, what is this module achieving that the existing module doesn't?

Reply:
Current module is used to get rbd mirror daemon status from both the clusters after rbd mirror deployment.
For same we have added new method "mirror_daemon_status" in "rbd_mirror_utils".